core/src/main/scala/ai/h2o/sparkling/backend/converters/SupportedRDD.scala

```diff
@@ -47,145 +48,164 @@ private[this] object SupportedRDD {
-      import hc.sparkSession.implicits._
+      val spark = SparkSessionUtils.active
+      import spark.implicits._
       SparkDataFrameConverter.toH2OFrameKeyString(hc, rdd.toDF(), frameKeyName, converter)
     }
   }
 
   implicit def toH2OFrameFromRDDLong(rdd: RDD[Long]): SupportedRDD = new SupportedRDD {
     override def toH2OFrameKeyString(hc: H2OContext, frameKeyName: Option[String], converter: Converter): String = {
-      import hc.sparkSession.implicits._
+      val spark = SparkSessionUtils.active
+      import spark.implicits._
       SparkDataFrameConverter.toH2OFrameKeyString(hc, rdd.toDF(), frameKeyName, converter)
     }
   }
 
   implicit def toH2OFrameFromRDDString(rdd: RDD[String]): SupportedRDD = new SupportedRDD {
     override def toH2OFrameKeyString(hc: H2OContext, frameKeyName: Option[String], converter: Converter): String = {
-      import hc.sparkSession.implicits._
+      val spark = SparkSessionUtils.active
+      import spark.implicits._
       SparkDataFrameConverter.toH2OFrameKeyString(hc, rdd.toDF(), frameKeyName, converter)
     }
   }
 
   implicit def toH2OFrameFromRDDLabeledPoint(rdd: RDD[LabeledPoint]): SupportedRDD = new SupportedRDD {
     override def toH2OFrameKeyString(hc: H2OContext, frameKeyName: Option[String], converter: Converter): String = {
-      import hc.sparkSession.implicits._
+      val spark = SparkSessionUtils.active
+      import spark.implicits._
       SparkDataFrameConverter.toH2OFrameKeyString(hc, rdd.toDF(), frameKeyName, converter)
     }
   }
 
   implicit def toH2OFrameFromRDDTimeStamp(rdd: RDD[java.sql.Timestamp]): SupportedRDD = new SupportedRDD {
     override def toH2OFrameKeyString(hc: H2OContext, frameKeyName: Option[String], converter: Converter): String = {
-      import hc.sparkSession.implicits._
+      val spark = SparkSessionUtils.active
+      import spark.implicits._
       SparkDataFrameConverter.toH2OFrameKeyString(hc, rdd.map(v => Tuple1(v)).toDF(), frameKeyName, converter)
     }
   }
 
   implicit def toH2OFrameFromRDDProductNoTypeTag(rdd: RDD[Product]): SupportedRDD = new SupportedRDD {
     override def toH2OFrameKeyString(hc: H2OContext, frameKeyName: Option[String], converter: Converter): String = {
-      SparkDataFrameConverter.toH2OFrameKeyString(hc, hc.sparkSession.createDataFrame(rdd), frameKeyName, converter)
+      SparkDataFrameConverter.toH2OFrameKeyString(hc, SparkSessionUtils.active.createDataFrame(rdd), frameKeyName, converter)
     }
   }
 
   implicit def toH2OFrameFromRDDProduct[A <: Product : ClassTag : TypeTag](rdd: RDD[A]): SupportedRDD = new SupportedRDD {
     override def toH2OFrameKeyString(hc: H2OContext, frameKeyName: Option[String], converter: Converter): String = {
-      SparkDataFrameConverter.toH2OFrameKeyString(hc, hc.sparkSession.createDataFrame(rdd), frameKeyName, converter)
+      SparkDataFrameConverter.toH2OFrameKeyString(hc, SparkSessionUtils.active.createDataFrame(rdd), frameKeyName, converter)
```

Comment:
nit: Can we extract `SparkSessionUtils.active.createDataFrame(rdd)` to a separate line?